pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -22,15 +22,20 @@
 }:
 
 let
-  llvmPackages = llvmPackages_11;
+  upstream-info = (lib.importJSON ./upstream-info.json).${channel};
+
+  llvmPackages = llvmPackages_11.override {
+    buildLlvmTools = llvmPackages.tools;
+    targetLlvmLibraries = llvmPackages.libraries;
+    buildOverride = upstream-info.deps.clang;
+  };
+
   stdenv = llvmPackages.stdenv;
```

Comment:
I probably should have set `stdenv = llvmPackages.lldClang.stdenv` in https://github.com/NixOS/nixpkgs/pull/101467. Maybe it would be good to include?

Reply:
Never mind, `llvmPackages.lldClang.stdenv` is totally broken.